deck.gl__mesh-layers/index.d.ts

```diff
@@ -53,19 +53,25 @@ declare module '@deck.gl/mesh-layers/simple-mesh-layer/simple-mesh-layer' {
     normals: Float32Array;
     texCoords: Float32Array;
   }
+  export interface Material {
+    ambient?: number;
+    diffuse?: number;
+    shininess?: number;
+    specularColor?: [number, number, number];
+  }
   export interface SimpleMeshLayerProps<D> extends LayerProps<D> {
-    mesh: SimpleMesh;
+    mesh: SimpleMesh | null;
```

Comment:
Are the `null`s necessary? Or can these be optional properties?

Reply:
According to the documentation null is allowed, but according to the code both undefined and null are allowed with the same behavior

Reply:
But `null` as a value is implicit, so do we need to specify the ` | null` ?

Reply:
maybe yes

Reply:
I'm a little confused. What should this type look like?

Reply:
I think it could be left untouched: 
```ts
mesh: SimpleMesh;
```
If the compiler is ok with that.

Reply:
In my case, typescript throws an error that I am passing null. But actually it's not an error for SimpleMesh.